tests/lib/rule.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import atexit
+import os
+import os.path
+import shlex
+import subprocess
+import sys
+
+import lib.oscap
+import lib.virt
+from lib.log import log
+
+
+DATA_DIR = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), 'data')
```

Comment:
It is pretty nasty to assume anything in sys.argv in a library. What you could do instead is rely on `__file__`. That way you are guaranteed that this will work regardless of who uses the library.